data/json/items/armor.json

```diff
@@ -2189,7 +2189,7 @@
     "type": "ARMOR",
     "name": "diver's watch",
     "name_plural": "diver's watches",
-    "description": "A waterproof, self-winding watch on a steel watchband.  Tells the time and current temperature and is impervious to water.  Activate to check the digital thermometer.",
+    "description": "A waterproof, self-winding watch on a stainless steel watchband.  Tells the time and current temperature.  Activate to check the digital thermometer.",
```

Comment:
stainless != impervious to water

Reply:
Nevermind, I didn't see there is already "waterproof" at the beginning.

Reply:
A regular steel watchband would be subject to intense rusting on a diver's watch. Most diver's watches use stainless steel to combat salt water.